examples/base/main.js

```diff
@@ -3,30 +3,35 @@ import path from 'path';
 import {foo} from './mod.js';
 
 function main(paths) {
-  return Promise.all(paths.map(function(name) {
-    return new Promise(function(resolve, reject) {
-      fs.readFile(name, function(err, data) {
-        if (err) {
-          reject(err);
-        } else {
-          resolve(String(data));
-        }
+  return Promise.all(
+    paths.map((name) => {
```

Comment:
We don't have to use arrow functions, but I wanted some examples of formatting where we do use them.